udata/core/topic/apiv2.py

```diff
@@ -157,6 +154,7 @@ def delete(self, topic):
         if not TopicEditPermission(topic).can():
             apiv2.abort(403, "Forbidden")
 
+        # TODO: should we ignore activity creation here?
```

Comment:
This will create a _lot_ of deleted activities (and probably crash since it is synchronous) if we empty a universe Topic. I'm not sure we should introduce an exception here to prevent that, WDYT?

Emptying a universe crashes anyway 😬 #3423